README.md

```diff
@@ -142,45 +142,42 @@ If you didn't use Raiden before, you can
 
 ### Try Out the Raiden Demo dApp
 
-These step-by-step instructions will guide you through the process for trying the Raiden dApp hosted at [https://lightclient.raiden.network/](https://lightclient.raiden.network/). We will be using the Ropsten testnet and MetaMask wallet in this example.
+These step-by-step instructions will guide you through the process for trying the Raiden dApp hosted at [https://lightclient.raiden.network/](https://lightclient.raiden.network/). We will be using the Goerli testnet and MetaMask wallet in this example.
 
 __Prerequisites__
 
-You need to have MetaMask installed for your browser.
+You need to have MetaMask installed for your browser. Visit [MetaMask's website](https://metamask.io/) to download and install MetaMask.
 
-1. Visit the [MetaMask website](https://metamask.io/) to download and install MetaMask.
-2. Select the Ropsten Test Network when you've logged in to your MetaMask.
+__Step 1: Acquire ETH__
+1. Visit the Goerli faucet at [htts://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog)
+2. Follow the instructions on the faucet's website to acquire Goerli ETH
 
+__Step 2: Acquire TTT (TestToken)__
+1. Visit [this page on Etherscan](https://goerli.etherscan.io/address/0x3ed0daedc3217615bde34fedd023bc81ae49251b#writeContract) where you can write to the TTT contract
+2. Enter your MetaMask and choose the Goerli account with ETH
+3. Go back to Etherscan and click "Connect to Web3"
+4. Scroll down to the "mint" field and enter ```1000000000000000000000```
+5. Click the "Write" button and confirm the transaction in MetaMask
 
```

Comment:
The user needs to check, if the transaction has been successful

Reply:
I added a line about successful transaction in the final bullet point. Also the gif I created yesterday shows this.